app/views/shared/_navbar.html.erb

```diff
@@ -8,8 +8,10 @@
           <li><%= link_to "Demo Nights", admin_demo_nights_path %></li>
           <% if !current_demo_night? %>
             <li><%= link_to "New Demo Night", new_admin_demo_night_path %></li>
+          <% else %>
+            <li><%= link_to "Current Projects", admin_demo_night_path(DemoNight.current) %></li>
           <% end %>
-          <li><%= link_to "Current Projects", admin_projects_path %></li>
```

Comment:
@notmarkmiranda One question: how crowded is the `_vote_talley.html.erb` partial looking? Noticed that on the show we include the module/`project_type`, but not on the DemoNight index. Not positive we'll need it, so if it makes things look sloppy we can leave it out. Otherwise do you think we add it?